index.js

```diff
@@ -6,112 +6,147 @@ continueDisplayDog.style.display = "none";
 var continueDisplayLlama = document.getElementById('continueLlama');
 continueDisplayLlama.style.display = "none";
 
+if (continueDisplayCat !== null) {
+    //then we ignore so the error is not thrown
+    continueDisplayCat.style.display = "none";
+}
+
+if (continueDisplayDog !== null) {
+    //then we ignore so the error is not thrown
+    continueDisplayDog.style.display = "none";
+}
+
+if (continueDisplayLlama !== null) {
+    //then we ignore so the error is not thrown
+    continueDisplayLlama.style.display = "none";
+}
+
+// TODO: Dog and llama too
```

Comment:
Looks like you accomplished this task above. so is this TODO done?

Reply:
Removed the TODO pseudocode